web_environment_ribbon/static/src/css/ribbon.css

```diff
@@ -14,7 +14,7 @@
     -moz-transform: rotate(-45deg);
     -o-transform: rotate(-45deg);
     transform: rotate(-45deg);
-    z-index: 1000;
+    z-index: 9999;
```

Comment:
The idea is to not put too much large texts. Anyway, you can offset the text a bit to let it uncovered, but make this thicker is not the solution, as it's still over the menu.